assets/js/phoenix_live_view.js

```diff
@@ -1134,6 +1134,7 @@ export class View {
   onLiveRedirect({to, kind}){
     this.href = to
     Browser.pushState(kind, {}, to)
+    this.liveSocket.registerNewLocation(window.location)
```

Comment:
@snewcomer I'm not sure if that is the proper fix though. Could you review that? maybe there is a better way to fix that 🙈

Reply:
Agreed.  Nice find and thank you for the gif repro!

Before we merge, I want to have this testable and/or in an example app.  I'll see if we can work through this by the end of the week.